### ISSUE TITLE
NASA discontinues anonymous FTP service.

### ISSUE THREAD

**Issue by Mictronics** [Mictronics/pluto-gps-sim#11]
The README.md suggests downloading the daily GPS ephemeris data from NASA's FTP server. This service is ending on October 31, 2020 as per their website. Please update the README to go suggest visiting the new HTTPS instead.

**Comment by cmconrad** [comment]
The new way to get ephemeris data is as follows:

1. Register an account with https://urs.earthdata.nasa.gov/ (There is no way around this step).
2. Create a file called `.netrc`  with the following [format](https://cddis.nasa.gov/Data_and_Derived_Products/CreateNetrcFile.html).
`machine urs.earthdata.nasa.gov login <username> password <password>` where \<username> and \<password> are the values you set when you created your Earthdata login account. Don't forget to `sudo chmod 004 .netrc` so that no one can read your file and get your username and password.

3. You can automate with the process with following script.
```
#!/bin/sh
day=$(date +%j)
year=$(date +%Y)
yr=$(date +%y)

curl -c /tmp/cookie -n -L -o "brdc""$day""0.$yr""n.Z" "https://cddis.gsfc.nasa.gov/archive/gnss/data/daily/$year""/brdc/brdc""$day""0.$yr""n.Z"

uncompress "brdc""$day""0.$yr""n.Z"
echo "brdc""$day""0.$yr""n.Z"

```

The `.netrc` file along with the `-n` flag allows curl to automatically authenticate when attempting to retrieve the file.

I hope this helps and please consider expanding the README to include more information like this.

**Comment by Mictronics** [comment]
Fixed with 769c2d6
